packages/fxa-settings/src/components/Avatar/index.stories.tsx

```diff
@@ -4,12 +4,17 @@
 
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { Avatar } from '.';
+import { MockedCache } from '../../models/_mocks';
```

Comment:
TIL! `MockedCache` is really clever

Reply:
Yeah, it's really nice! There's a note [in the docs about it here](https://github.com/mozilla/fxa/tree/main/packages/fxa-settings#components-that-use-useaccount). :)